src/molecules/CardContent/CardContent.jsx

```diff
@@ -1,13 +1,22 @@
+import { FolderKebab } from "../FolderKebab";
 import "./CardContent.css";
 
-export const CardContent = ({ elapsedTime, description, createdAt, isHovered }) => {
-  const className = isHovered ? "CardContent CardContent-hovered" : "CardContent";
+export const CardContent = ({
+  elapsedTime,
+  description,
+  created_at,
+  isHovered,
+}) => {
+  const className = isHovered
+    ? "CardContent CardContent-hovered"
+    : "CardContent";
```

Comment:
```suggestion
  const className = `CardContent ${isHovered ? "CardContent-hovered" : "" }`;
```
개인적으로는 이 방법을 더 선호하는 편입니다.

그리고 변수의 이름도 className 보다는 조금 더 특정할 수 있는 이름으로 지어주세요